Branch-SDK/src/io/branch/referral/ShareLinkManager.java

```diff
@@ -166,7 +166,7 @@ private void createShareDialog(List<SharingHelper.SHARE_WITH> preferredOptions)
             textView.setTextAppearance(context_, android.R.style.TextAppearance_Medium);
             textView.setTextColor(context_.getResources().getColor(android.R.color.darker_gray));
             textView.setPadding(leftMargin, padding, padding, padding);
-            shareOptionListView.addHeaderView(textView);
+            shareOptionListView.addHeaderView(textView, null, false);
```

Comment:
Sorry it is my bad. Didn't realize the list header is selectable  by default. The above change look good but we need to do the same on Line #160